src/Services/MeasurementConverterService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Furniture\Services;
+class MeasurementConverterService
+{
+    const VALID_UNITS = ['mm', 'cm', 'in', 'ft'];
+    const MM = 1;
+    const CM = 10;
+    const IN = 25.4;
+    const FT = 304.8;
+
+    public static function convertMeasurement(string $unit, int $value):float
```

Comment:
This looks great, but I think it would be convenient to include something like "from mm" in the method name to make it clear what you're converting from!